Rakefile

```diff
@@ -153,6 +153,13 @@ task "assets:precompile" do
   fail unless $?.success?
 end
 
+desc "Validate, lint, format OpenAPI YAML file"
+task :openapifmt do
+  sh "npx redocly lint openapi.yml"
+  sh "npx @stoplight/spectral-cli --fail-severity=warn lint openapi.yml"
+  sh "echo 'sortPathsBy: path' | npx -- openapi-format -o openapi.yml --sortFile /dev/stdin openapi.yml"
```

Comment:
Could you describe the function of each tool using just one sentence in the commit message?

Reply:
Sure.